api/peripheral_api/routes/Printer.js

```diff
@@ -88,4 +93,30 @@ router.post('/sendPrintRequest', async (req, res) => {
   });
 });
 
+router.post('/addFilePDF', async (req, res) => {
```

Comment:
rename it to something like validateDiscordReq or like incomingDiscordReq, just something that describes what the function does